rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -671,6 +671,33 @@ private static int calculateLegacyTxSize(Federation federation, int inputsCount,
         return baseSize + signingSize;
     }
 
+    public static int calculateSignedSegwitBtcTxVirtualSize(BtcTransaction btcTx) {
```

Comment:
```suggestion
    public static int calculateBtcTxVirtualSize(BtcTransaction btcTx) {
```
you are not using anywhere the segwit nor the signed part right?